packages/hawtio/src/plugins/camel/camel-service.ts

```diff
@@ -333,6 +335,7 @@ export function getCamelVersions(): string[] {
  * the given node. Currently, it supports Camel v3 and v4.
  */
 export function getCamelModel(node: MBeanNode): CamelModel {
+  const { camel4, camel3 } = fetchCamelModules()
```

Comment:
When running I get an error that camel3 is null

Reply:
Yeah, rather they should be `Promise` so that they will never return `null`.